rally_ovs/plugins/ovs/scenarios/ovn.py

```diff
@@ -251,6 +254,43 @@ def _create_networks(self, network_create_args):
         return lswitches
 
 
+    @atomic.action_timer("ovn_network.of_check_port")
+    def _of_check_ports(self, lports, sandboxes, port_bind_args):
```

Comment:
**Why?** In our OVN scalability test with dataplane (NOTE: not ovn-scale-test/sandbox), we found that some lport is not pingable after binding. After certain time period, the lport becomes pingable. One suspicious point is that the openflow rules after binding is not setup fast enough. Therefore checking "port state up" is insufficient.
**What?** In ovn-scale-test, since there is no real kernel module installed, the ping tool is not available. Therefore, we have to use OVS trace tool to verify the openflow rules.
**How?** Use the OVS built-in trace tool to verify the openflow rules are set correctly.